addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -326,6 +326,20 @@ body.editor_enable.editor_has_snippets {
                 // FIXME temporary fix for m2o option for example
                 position: static !important;
             }
+
+            > we-alert {
+                background-color: var(--o-we-snippets-bg-info);
+                display: block;
+                margin: -.5em;
```

Comment:
Negative margin for this feature? Strange?

Reply:
`we-customizeblock-options` has a padding of .5em and I want my alert to overlap that padding. Maybe there is better way to do it but I do not know it...

Reply:
I understood that, ok not important anyway, the design will be reviewed.